ui/react-components/components/AppointmentEditorFooter/AppointmentEditorFooter.jsx

```diff
@@ -31,26 +37,30 @@ const AppointmentEditorFooter = props => {
             <div className={classNames(footerElements)}>
                 <CustomPopup triggerComponent={cancelButton} popupContent={popupContent} style={customPopup}/>
                 {isEdit
```

Comment:
Update button is a completely different kind of button, why do we want to club it with the existing save button?

Reply:
In few cases, we don't need dropdown view for update(like start date change and end date change for recurring appointment). For the above given cases, just single update button is enough. So for few changes, we can use the existing component. Code chunk can be extracted but there won't be any logical/functional difference.